lib/live_deck_web/live/control_view.ex

```diff
@@ -0,0 +1,25 @@
+defmodule LiveDeckWeb.ControlView do
+  use Phoenix.LiveView
+  alias LiveDeck.Presentations
+  require Logger
+  @slides Presentations.list_slides()
+
+  def mount(_params, _session, socket) do
+    {:ok, assign(socket, [current: 0, slides: @slides])}
+  end
+
+  def render(assigns) do
+    ~L"""
+      <button phx-click="prev">Previous!</button>
+      <button phx-click="next">Next!</button>
+    """
+  end
+
+  def handle_event("next", _, socket) do
```

Comment:
We won't have to set `slides: @slides` here. The socket will carry around the initial state that we set in the mount callback.